src/classBased/Components/TodosList.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable */
+import React from 'react';
+import TodoItem from './TodoItem';
+
+class TodosList extends React.Component {
```

Comment:
Please try not to disable linters, it would be better if you fix linter errors instead